mitmproxy/tcp.py

```diff
@@ -47,5 +50,10 @@ def __init__(self, client_conn, server_conn, live=None):
         messages=List[TCPMessage]
     )
 
+    def copy(self):
```

Comment:
This should not be required, the [default implementation](https://github.com/mitmproxy/mitmproxy/blob/fa661217c12733da4afe500b51a253f5994f3506/mitmproxy/types/serializable.py#L32) should work?